core/common/src/thrift/file_system_master.thrift

```diff
@@ -251,7 +256,9 @@ service FileSystemMasterClientService extends common.AlluxioService {
    */
   void remove(
     /** the path of the file or directory */ 1: string path,
+    // This is deprecated since 1.5 and will be removed in 2.0. Use DeleteTOptions.
     /** whether to remove recursively */ 2: bool recursive,
+    /** the options for deleting the file */ 3: DeleteTOptions options,
```

Comment:
Is this change backward compatible? (1.4 client can talk to 1.5 master, and 1.5 client can talk to 1.4 master)